h2o-algos/src/main/java/hex/schemas/GLMV3.java

```diff
@@ -277,11 +287,20 @@ public static final class GLMParametersV3 extends ModelParametersSchemaV3<GLMPar
     @API(help="Request p-values computation, p-values work only with IRLSM solver and no regularization", level = Level.secondary, direction = Direction.INPUT)
     public boolean compute_p_values; // _remove_collinear_columns
 
+    @API(help="If true, will fix tweedie variance power value to the value set in tweedie_variance_power",
+            level=Level.secondary, direction=Direction.INPUT)
+    public boolean fix_tweedie_variance_power;
+
     @API(help="In case of linearly dependent columns, remove some of the dependent columns", level = Level.secondary, direction = Direction.INPUT)
     public boolean remove_collinear_columns; // _remove_collinear_columns
 
     @API(help = "if changes in dispersion parameter estimation is smaller than dispersion_epsilon, will break out of the dispersion parameter estimation loop using maximum likelihood", level = API.Level.secondary, direction = API.Direction.INOUT)
     public double dispersion_epsilon;
+
+    @API(help = "In estimating tweedie dispersion parameter using maximum likelihood, this is used to choose the lower" +
+            " and upper indices in the approximating of the infinite series summation.", 
+            level = API.Level.secondary, direction = API.Direction.INOUT)
+    public double tweedie_epsilon;
     
     @API(help = "control the maximum number of iterations in the dispersion parameter estimation loop using maximum likelihood", level = API.Level.secondary, direction = API.Direction.INOUT)
```

Comment:
It wasn't changed in this PR but it's just a small change in favor of consistency.  (Feel free to ignore this.)
```suggestion
    @API(help = "Control the maximum number of iterations in the dispersion parameter estimation loop using maximum likelihood.", level = API.Level.secondary, direction = API.Direction.INOUT)
```